### ISSUE TITLE
Add a imprint page and link it in navbar and footer

### ISSUE THREAD

**Issue by Abwasserrohr** [OdyseeTeam/odysee-frontend#1401]
**Is your feature request related to a problem? Please describe.**
For sites that also target a German audience, it's required these sites to have a imprint in German.

**Describe the solution you'd like**
A imprint site (similar to the terms site) that is linked from the navbar and footer.
It should be possible to translate the contents of the page that contains the contact info like address, email and tax info.
This is how it looks at [YT](https://www.youtube.com/t/impressum?hl=de&gl=DE).

**Additional context**
![](https://user-images.githubusercontent.com/44294954/165292506-7e150e2a-bc15-4b3a-b9c5-b2cdfa287781.png)

**Comment by tzarebczan** [comment]
So just copy/paste that text and replace odysee stuff?

**Comment by Abwasserrohr** [comment]
Yeah, a copy/paste of the terms page just with different text is required, I can provide that.
